### ISSUE TITLE
ControlNet OpenPose Issue: Missing required arguments: 'detect_hand', 'detect_body', and 'detect_face'

### ISSUE THREAD

**Issue by beltonk** [Stability-AI/StableSwarmUI#169]
I tried OpenPosePreprocessor / DWPreprocesser x Stock / t2i / thibaud models, all failed with the following error messages:

**"No images were generated (all refused, or failed - check server logs for details)."**

**Logs:**

1. OpenPose

got prompt
ERROR:root:!!! Exception during processing !!!
ERROR:root:Traceback (most recent call last):
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 153, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 83, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 76, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
TypeError: OpenPose_Preprocessor.estimate_pose() missing 3 required positional arguments: 'detect_hand', 'detect_body', and 'detect_face'

2. DWPose

got prompt
ERROR:root:!!! Exception during processing !!!
ERROR:root:Traceback (most recent call last):
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 153, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 83, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "/usr/local/src/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 76, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
TypeError: DWPose_Preprocessor.estimate_pose() missing 3 required positional arguments: 'detect_hand', 'detect_body', and 'detect_face'

**Comment by mcmonkey4eva** [comment]
This is technically a bug in the DWPreprocessor node rather than swarm - it has optional inputs, but the actual code requires those inputs, and so it's erroring out. This is easy to compensate for on swarm's side without side effects though so I went ahead and did it anyway (it now emits values even for the explicitly optional parameters).

**Comment by beltonk** [comment]
How about OpenPosePreprocessor? Both of them fails. In other words OpenPose cannot be used with Swarm UI...

**Comment by beltonk** [comment]
Thank you! Just updated and retried both DWPreprocessor & OpenposePreprocessor, they both work now!

One thing I find it weird is that once I apply OpenPose in ControlNet, the output image will tend to look fake and oldish 3D. I wonder why the style is affected with OpenPose ControlNet. Does resolution matter? 

Is ControlNet Strength 1 in SwarmUI == Control Weight 1 in A111?

Can I assume OpenPose in SwarmUI with have similar results in A1111's default settings:
- Starting Control Step: 0
- Ending Control Step: 1
- Control Mode: Balanced
- Preprocessor Resolution: 512

In other words, what's the default ControlNet behaviour in SwarmUI GUI as they are not configurable?

**Comment by mcmonkey4eva** [comment]
`Is ControlNet Strength 1 in SwarmUI == Control Weight 1 in A111?` yes
`Starting Control Step: 0
Ending Control Step: 1` yes
`Control Mode: Balanced` Not sure what that actually is. It doesn't have a corresponding comfy param.
`Preprocessor Resolution: 512` images are fed in at whatever resolution you have. Some preprocessors implicitly modify scale where they need to (that's more a question for the comfy preproc extension)
